bitwarden_license/src/Sso/Controllers/AccountController.cs

```diff
@@ -235,37 +238,74 @@ public IActionResult ExternalChallenge(string scheme, string returnUrl, string s
     [HttpGet]
     public async Task<IActionResult> ExternalCallback()
     {
+        // Feature flag (PM-24579): Prevent SSO on existing non-compliant users.
+        var preventNonCompliant = _featureService.IsEnabled(FeatureFlagKeys.PM24579_PreventSsoOnExistingNonCompliantUsers);
```

Comment:
Nit (naming):  `preventNonCompliant` doesn't actually tell us what the user is not compliant with. We are preventing org users from logging in if they are in an invalid status (invited or revoked). Something like `preventOrgUserLoginIfStatusInvalid` would be much easier to reason about